examples/step-68/doc/results.dox

```diff
@@ -0,0 +1,90 @@
+<h1>Results</h1>
+
+The directory in which this program is run contains an example parameter file by default.
```

Comment:
This is not true at the moment, please add the example prm file.

Reply:
Is this still an issue? If not, please "resolve"